modules/UI/UI.js

```diff
@@ -1140,11 +1140,12 @@ UI.notifyFocusLeft = function () {
  * @param {string} [login] current login
  */
 UI.updateAuthInfo = function (isAuthEnabled, login) {
+    let showAuth = isAuthEnabled && UIUtil.isAuthenticationEnabled();
```

Comment:
Ok, now I see that you take isAuthEnabled into account actually, but also use the UI property. Why do we need the UI property then?